Doc/library/io.rst

```diff
@@ -1043,6 +1043,21 @@ Text I/O
       .. versionchanged:: 3.11
          The method supports ``encoding="locale"`` option.
 
+   .. method:: seek(cookie, whence, /)
+
+      Set the stream position, and return the new stream position.
+
```

Comment:
Given that the return type isn't useful for ``TextIOWrapper``, perhaps just don't put what it returns in the summary line? The suggested returns/rtype pair could even be at the very end.

```suggestion
      Set the stream position.
      
      :returns: The new stream position.
      :rtype: int
```

Reply:
I think that renders a bit strange, since we're not using a parameter list (which I don't think is needed for this method). I'd prefer something like this:

```suggestion
      Set the stream position.
      Return the new stream position as an :class:`int`.
```